### ISSUE TITLE
[Bug]: Last update breaks `cnv` workflow

### ISSUE THREAD

**Issue by bounlu** [epi2me-labs/wf-human-variation#51]
### What happened?

After the last update, `cnv:callCNV` workflow fails with the attached log error with nextflow.

When I log into the docker container myself, I can confirm `ezcharts` is not imported by python although it's installed via conda:

```
docker run --rm -it ontresearch/wf-cnv:sha428cb19e51370020ccf29ec2af4eead44c6a17c2 bash

conda list | grep ezcharts
ezcharts                  0.3.6                      py_0    epi2melabs

epi2melabs@56eb211ab9c6:~$ python
Python 3.8.15 | packaged by conda-forge | (default, Nov 22 2022, 08:46:39) 
[GCC 10.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.

>>> from ezcharts.components.fastcat import SeqSummary
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'ezcharts'

>>> import ezcharts
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'ezcharts'
```

### Operating System

ubuntu 20.04

### Workflow Execution

Command line

### Workflow Execution - CLI Execution Profile

Docker

### Workflow Version

latest

### Relevant log output

```shell
ERROR ~ Error executing process > 'cnv:callCNV (1)'

Caused by:
  Essential container in task exited

Command executed:

  run_qdnaseq.r --bam calls_align_sort_combined.bam --out_prefix NA12878 --binsize 500 --reference hg38
  cut -f5 NA12878_calls.bed | paste NA12878_bins.bed - > NA12878_combined.bed
  check=`awk -F "	" 'NF != 6' NA12878_segs.seg`
  
  if [ -n "${check}" ]; then             echo "vcf is malformed";             workflow-glue fix_vcf --vcf NA12878_calls.vcf --fixed_vcf NA12878_calls_fixed.vcf --sample_id NA12878;             workflow-glue fix_vcf --vcf NA12878_segs.vcf --fixed_vcf NA12878_segs_fixed.vcf --sample_id NA12878; 
      rm NA12878_calls.vcf
      mv NA12878_calls_fixed.vcf NA12878_calls.vcf
      rm NA12878_segs.vcf
      mv NA12878_segs_fixed.vcf NA12878_segs.vcf
  fi

Command exit status:
  1

Command output:
  [1] "NA12878_raw_bins.bed"
  [1] "NA12878_bins.bed"
  [1] "NA12878_calls.bed"
  [1] "NA12878_calls.vcf"
  [1] "NA12878_segs.seg"
  [1] "NA12878_segs.bed"
  [1] "NA12878_segs.vcf"
  vcf is malformed

Command error:
  (more omitted..)
  5,760	of which in selected chromosomes
  5,549	of which with reference sequence
  4,678	final bins
  [1] "NA12878_raw_bins.bed"
  Plotting sample calls_align_sort_combined median read counts
  Calculating correction for GC content and mappability
      Calculating fit for sample calls_align_sort_combined (1 of 1) ...
  Done.
  Note: Residual filter missing for chromosomes: X, Y
  6,188	total bins
  6,188	of which in selected chromosomes
  5,918	of which with reference sequence
  5,005	final bins
  Note: Filtering out additional 27 bins due to missing values.
  Applying median normalization ...
  Smoothing outliers ...
  Plotting sample calls_align_sort_combined (176,502,708 reads) (1 of 1) ...
  [1] "NA12878_bins.bed"
  Performing segmentation:
      Segmenting: calls_align_sort_combined (1 of 1) ...
  Plotting sample calls_align_sort_combined (176,502,708 reads) (1 of 1) ...
  Calling aberrations with the following cutoffs:
  homozygous deletion < -2 < loss < -0.42 < normal < 0.32 gain < 2.32 < amplification
  Plotting sample calls_align_sort_combined (176,502,708 reads) (1 of 1) ...
  Plotting sample calls_align_sort_combined (176,502,708 reads) (1 of 1) ...
  Plotting sample calls_align_sort_combined median read counts
  [1] "NA12878_calls.bed"
  [1] "NA12878_calls.vcf"
  [1] "NA12878_segs.seg"
  [1] "NA12878_segs.bed"
  [1] "NA12878_segs.vcf"
  vcf is malformed
  Traceback (most recent call last):
    File "/home/epi2melabs/nextflow-bin/workflow-glue", line 7, in <module>
      cli()
    File "/home/epi2melabs/nextflow-bin/workflow_glue/__init__.py", line 51, in cli
      f'{_package_name}.{comp}' for comp in get_components()]
    File "/home/epi2melabs/nextflow-bin/workflow_glue/__init__.py", line 22, in get_components
      mod = importlib.import_module(f"{_package_name}.{name}")
    File "/home/epi2melabs/conda/lib/python3.8/importlib/__init__.py", line 127, in import_module
      return _bootstrap._gcd_import(name[level:], package, level)
    File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
    File "<frozen importlib._bootstrap>", line 991, in _find_and_load
    File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
    File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
    File "<frozen importlib._bootstrap_external>", line 843, in exec_module
    File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
    File "/home/epi2melabs/nextflow-bin/workflow_glue/report_al.py", line 5, in <module>
      from ezcharts.components.fastcat import SeqSummary
  ModuleNotFoundError: No module named 'ezcharts'
```

**Comment by SamStudio8** [comment]
@bounlu Thanks for taking the time to report this, I think I know what's going on and aim to deploy a fix this week.

**Comment by SamStudio8** [comment]
@bounlu This should now be fixed in wf-human-variation 1.6.1, please use `nextflow pull epi2me-labs/wf-human-variation` to update your copy. Let me know if this solves your issue. Thanks again for reporting!

**Comment by bounlu** [comment]
@SamStudio8 Thanks for this. I am testing it now.
